gh-actions/go/generate/action.yaml

```diff
@@ -24,14 +24,28 @@ outputs:
 runs:
   using: "composite"
   steps:
+    - name: Set up jq
+      shell: bash
+      run: |
+        echo "::group::Download jq"
```

Comment:
Please set -eu on the script, like all others, to ensure we stop early if we can’t update/install jq.

Reply:
Oh, weird, I was guess that it was enabled by default (iirc it is in gitlab scripts...).